### PR TITLE
Add tests for Histogram and CentralityClasses

### PR DESCRIPTION
**Pull request by Hendrik1704**
This adds unit tests for the `Histogram` class and fixes issue #184. I added another `raise ValueError` in the function to scale histograms.

### CODE REVIEW

File: tests/test_CentralityClasses.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import pytest
+import numpy as np
+from sparkx.CentralityClasses import CentralityClasses
+
+
```
Comment (NGoetz) [inline tests/test_CentralityClasses.py:5]: What happens if I add centrality bins with a value greater 100? Should this be tested?

File: tests/test_Histogram.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    hist = Histogram((0, 4, 4))
+    hist.add_value([1, 2, 3])
+    hist.statistical_error()
+    hist_labels = [{'bin_center': 'Bin Center', 'bin_low': 'Bin Low', 'bin_high': 'Bin High',
+                    'distribution': 'Distribution', 'stat_err+': 'Stat Error+', 'stat_err-': 'Stat Error-',
+                    'sys_err+': 'Sys Error+', 'sys_err-': 'Sys Error-'}]
+    filename = tmp_path / "test_histograms.csv"
+    hist.write_to_file(filename, hist_labels)
+
+    print(hist.histogram())
```
Comment (NGoetz) [inline tests/test_Histogram.py:81]: Should tests really print out every time they run?

**Review by NGoetz — APPROVED** [review]
I approve this PR, but have a look at both comments I added.

File: tests/test_Histogram.py (hunk 2 above)
Comment (Hendrik1704) [reply tests/test_Histogram.py:81]: You are completely right. This was a test of the test leftover.

File: tests/test_CentralityClasses.py (hunk 1 above)
Comment (Hendrik1704) [reply tests/test_CentralityClasses.py:5]: I'll add more tests in the constructor.

**Pull request merged**